k8s/postgres.yaml

```diff
@@ -44,6 +44,7 @@ spec:
         app: pupperware
         svc: postgres
     spec:
+      hostname: postgres.internal
```

Comment:
As @jarretlavallee discovered, k8s doesn't like hostnames with a `.` in them, so we're going to have to put up another PR to address the subdomain configuration we're striving for across the board.